com.github.hopsan.Hopsan.desktop

```diff
@@ -0,0 +1,8 @@
+[Desktop Entry]
+Version=1.0
+Type=Application
+Name=HopsanGUI
+Comment=The Hopsan simulation program
+Icon=com.github.hopsan.Hopsan
+Exec=hopsangui
+Categories=Science;Engineering;Education;
```

Comment:
Having keywords also here could make searching for the app easier.

Reply:
Yes, added some keywords